Makefile

```diff
@@ -577,6 +577,11 @@ ifeq ($(dot-config),1)
 # oldconfig if changes are detected.
 -include include/config/auto.conf.cmd
 
+# Read in DTS derived configuration, if it exists
+ifneq ($(ARCH),)
```

Comment:
why the test for $(ARCH) being set?  Probably worth adding a bit more to the comment about this